skyportal/tests/frontend/test_shifts.py

```diff
@@ -112,9 +113,11 @@ def test_shift(
     driver.wait_for_xpath(add_users_button)
     driver.click_xpath(add_users_button)
 
-    # check for button to add and remove users
     remove_users_button = '//*[@id="remove-users-button"]'
-    driver.wait_for_xpath(remove_users_button)
+    # waith for xpath to stay for a second to allow the button to be clickable
+    driver.wait_for_xpath(remove_users_button, timeout=1)
+    time.sleep(1)
+    driver.wait_for_xpath(remove_users_button, timeout=1)
```

Comment:
```suggestion
    time.sleep(1)
```
Seems to me like you can just put the wait and then click (I believe click_xpath also does a wait_for_xpath internally). 
Could be worth a shot. 
In any case, add a one-line comment why you add the wait time